ui/perfherder/helpers.js

```diff
@@ -434,8 +434,14 @@ export const getTextualSummary = (alerts, alertSummary, copySummary = null) => {
 
   const formatAlert = (alert, alertList) => {
     const numFormat = '0,0.00';
+    let amountPct;
 
-    const amountPct = alert.amount_pct.toFixed(0).padStart(3);
+    if (alert.amount_pct.toFixed(0) === '0') {
+      // have two fraction digits for the amountPct when it is rounded to 0
```

Comment:
Just a slight suggestion: I think we could avoid the `amountPct` mentioning & make the comment a bit more generic. For example:
`// have extra fraction digits when rounding ends up with 0%`

This looks good!

Reply:
@camd could you also take a look over this change?
I tested this locally & behaves accordingly.